README.md

```diff
@@ -19,39 +19,27 @@ The all-in-one starter kit for high-performance SaaS applications. With a few cl
+Once the project has been deployed, continue with the configuration steps below.
 
-In your Supabase dashboard, go to the SQL editor. There, the welcome tab has a "Quick Start" section. Select the "Stripe Subscriptions" quick start (it has the same content as the [`schema.sql` file](./schema.sql)) and hit the "RUN" button.
+## Configure Supabase Auth
 
 #### [Optional] - Set up OAuth providers
 
 You can use third-party login providers like GitHub or Google. Refer to the [docs](https://supabase.io/docs/guides/auth#third-party-logins) to learn how to configure these.
 
-### 3. Get your Supabase credentials
-
-In your [Supabase Dashboard](https://app.supabase.io/), go to the Project Settings (the cog icon), open the API tab, and find your API URL, the public `anon` key, and the secret `service_role` key. You will be prompted for these when deploying with Vercel.
-
-## Deploy with Vercel
-
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fthorwebdev%2Fnextjs-subscription-payments&env=NEXT_PUBLIC_SUPABASE_URL,NEXT_PUBLIC_SUPABASE_ANON_KEY,SUPABASE_SERVICE_ROLE_KEY&envDescription=Add%20your%20API%20keys%20from%20the%20Supabase%20Dashboard&project-name=nextjs-subscription-payments&repo-name=nextjs-subscription-payments&demo-title=Next.js%20Subscription%20Payments%20Starter&demo-description=Demo%20project%20on%20Vercel&demo-url=https%3A%2F%2Fnextjs-subscription-payments-starter.vercel.app%2F&demo-image=https%3A%2F%2Fnextjs-subscription-payments-starter.vercel.app%2Fdemo.png&integration-ids=oac_pb1dqJT8Ry2D99Q0o9qXWIhJ&external-id=nextjs-subscription-payments)
-
-Once your project has been deployed, continue with the configuration steps below. Note that this deployment step includes prompts for automatically creating a webhook endpoint for you.
-
-### Configure Supabase Auth
-
-After deploying, copy the deployment URL and navigate to your Supabase project settings (Authentication > Settings) and set your site url.
-
 ## Configure Stripe
 
 ### Create product and pricing information
 
-For Stripe to automatically bill your users for recurring payments, you need to create your product and pricing information in the [Stripe Dashboard](https://dashboard.stripe.com/test/products). When you create or update your product and price information, the changes are automatically synced with your Supabase database, as long as the webhook is configured correctly (the webhook creation is part of deploying to Vercel, the webhook endpoint is configured at the `/api/webhooks` path).
+For Stripe to automatically bill your users for recurring payments, you need to create your product and pricing information in the [Stripe Dashboard](https://dashboard.stripe.com/test/products). When you create or update your product and price information, the changes are automatically synced with your Supabase database, as long as you've selected "Set up webhooks" during the Vercel deployment.
```

Comment:
Considering we're in test mode, it'd be awesome if the integration automatically set up the product / pricing to match this demo. Then, they can edit them before going live.

The ideal state is that the application is 100% working after you deploy.

Reply:
That's tricky, because for the product sync to work we need a) the database to be ready and b) the webhook API route to be deployed. 

I could add a fixture folks could run via the Stripe CLI to create the products after the deployment is completed, but at that point it might be easier to just create a product in the Dashboard. The app nowadays sends you directly to the Dashboard:
![image](https://user-images.githubusercontent.com/5748289/102101812-0f58db80-3e66-11eb-9dab-2ba693213973.png)